tests/components/UserActions/ClickedDocumentList.spec.ts

```diff
@@ -117,6 +118,54 @@ describe('ClickedDocumentList', () => {
         });
     });
 
+    it('should not show the same query twice', () => {
```

Comment:
This test is quite long, maybe put comment to indicate where are the phases of the tests and explain what you're doing.